lib/linear_percent_indicator.dart

```diff
@@ -170,12 +170,11 @@ class _LinearPercentIndicatorState extends State<LinearPercentIndicator>
     WidgetsBinding.instance.addPostFrameCallback((_) {
       if (mounted) {
         setState(() {
-          _containerWidth = _containerKey.currentContext?.size?.width ?? 0.0;
-          _containerHeight = _containerKey.currentContext?.size?.height ?? 0.0;
+          _containerWidth = (_containerKey.currentContext?.size != null) ? _containerKey.currentContext!.size!.width : 0.0;
```

Comment:
can you revert these changes? I think this is not the root of the problem

Reply:
fixed in the last commit 🫡